virtual_rainforest/models/hydrology/hydrology_model.py

```diff
@@ -390,7 +444,7 @@ def update(self, time_index: int) -> None:
             relative_humidity=subcanopy_humidity,
             atmospheric_pressure=subcanopy_pressure,
             soil_moisture=soil_moisture_infiltrated,
-            wind_speed=0.1,  # TODO wind_speed,
+            wind_speed=0.1,  # TODO wind_speed in data object,
```

Comment:
Is this in m/s or something else? Might be worth mentioning the units in the comment